delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/StorageFileRejection.scala

```diff
@@ -138,6 +137,14 @@ object StorageFileRejection {
           s"File cannot be saved because it already exists on path '$path'."
         )
 
+    /**
+      * Rejection returned when a file can not be saved because content-length is not provided
+      */
+    final case object ContentLengthIsMissing
```

Comment:
When doing a put with a stream and asking for a checksum, it seems that the content length is needed